packages/orchestration/src/orchestration.js

```diff
@@ -188,18 +218,18 @@ const prepareOrchestration = (zone, createChainAccount) =>
       self: {
         async bindPort() {
           const network = getPower(this.state.powers, 'network');
-          const port = await E(network)
-            .bind(`/ibc-port/icacontroller-${this.state.icaControllerNonce}`)
-            .catch(e => Fail`Failed to bind port: ${e}`);
+          const port = await E(network).bind(
```

Comment:
out of scope but it strikes me that `.bind(` is 99% of the time the JS function bind to `this`.

@iomekam WDYT of a name that's less likely to be confused?

Reply:
Maybe `bindPort`? https://github.com/Agoric/agoric-sdk/blob/80e450a9afe0441406fa3e020d94ef0af6366851/golang/cosmos/x/vibc/types/receiver.go#L146

I can see how `E(port).bindPort()` might be sound odd, though.

Reply:
redundant > confusing ;-)

Reply:
`bindPort` is a perfect name.  It's not `E(port).bindPort()`, it's `E(network).bindPort()`.